ssas/service/main/main.go

```diff
@@ -107,6 +116,10 @@ func main() {
 		listExpiringCredentials()
 		return
 	}
+	if doGetXdata && (clientID != "" || APIKey != "") {
+		getXData(clientID, APIKey)
```

Comment:
Would it make sense to return an error if the caller does not supply the clientID or APIKey?